driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/ClientSessionHelper.java

```diff
@@ -56,30 +48,10 @@ Mono<ClientSession> withClientSession(@Nullable final ClientSession clientSessio
     }
 
     Mono<ClientSession> createClientSessionMono(final ClientSessionOptions options, final OperationExecutor executor) {
```

Comment:
@rozza would it be worth it change this to no longer return a `Mono`, now that it doesn't do I/O?  I would have done it but couldn't quite figure out how to change the calling code in `com.mongodb.reactivestreams.client.internal.OperationExecutorImpl#execute`

Reply:
I think we'd still want the protection from the Mono to handle any error scenarios - eg different MongoClients originating the session.